tests/modules/nf-core/cellranger/count/test.yml

```diff
@@ -63,3 +59,4 @@
     - path: output/cellranger/sample-123/outs/raw_feature_bc_matrix/matrix.mtx.gz
       md5sum: bdce94a51f16e22d40301724080b76ee
     - path: output/cellranger/sample-123/outs/web_summary.html
+      md5sum: edfa1a0dc666c38f9740167c045fb3c8
```

Comment:
Yeah that one is still not passing, should ditch md5 and just search for an exemplary output line as for others using `contain` if this is possible.

Reply:
I updated tests to use "contain" as suggested and confirmed they pass running locally.

```bash
$ nextflow run ./tests/modules/nf-core/universc/count -entry test_universc_count -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/universc/count/nextflow.config
N E X T F L O W  ~  version 22.04.5
Launching `./tests/modules/nf-core/universc/count/main.nf` [big_northcutt] DSL2 - revision: c3a0708ae3
executor >  local (3)
[ad/1d3f88] process > test_universc_count:UNIVERSC_MKGTF (genome.gtf)   [100%] 1 of 1 ?
[f1/ead2e2] process > test_universc_count:UNIVERSC_MKREF (genome.fasta) [100%] 1 of 1 ?
[da/0aa7e9] process > test_universc_count:UNIVERSC_COUNT (123)          [100%] 1 of 1 ?
Completed at: 18-Nov-2022 10:16:49
Duration    : 6m 20s
CPU hours   : 0.2
```